apps/web/src/components/forms/monitor/section-requests.tsx

```diff
@@ -39,7 +39,7 @@ export function SectionRequests({ form, type }: Props) {
       form.resetField("statusAssertions");
       form.resetField("textBodyAssertions");
     }
-  }, [type, jobType]);
+  }, [type, form.resetField]);
```

Comment:
Why?

Reply:
required dependancies 
jobType is not used in the hook, and form.resetField was missing